### PR TITLE
feat: add geocoding utility (#275)

### LINKED ISSUE #275: Integrate Google Geocoding API
Add a service in action server to obtain GPS coordinates from postal code/address.

### PR DESCRIPTION
**Pull request by mathieubergeron**
## Description
Added geocoding facility 

```
Geocoding().get("h0h0h0") 
# (45.678, -75.123)
```

## Related issues
closes #275 

## Checklist

- [x] PR title follows [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) guidelines 
- [x] All relevant PR sections are populated, irrelevant ones are removed

### CODE REVIEW

File: action-server/tests/utils/test_geocoding.py [hunk 1]
```diff
@@ -0,0 +1,59 @@
+from unittest import TestCase
+from unittest.mock import patch
+
+from covidflow.utils.geocoding import Geocoding
+
+ENV = {"GOOGLE_GEOCODING_API_KEY": "123abc"}
+
+POSTAL_CODE = "H0H0H0"
```
Comment (fparga) [inline action-server/tests/utils/test_geocoding.py:8]: :santa:

Comment (mathieubergeron) [reply action-server/tests/utils/test_geocoding.py:8]: ouais, désolé :P

File: action-server/covidflow/utils/geocoding.py [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+GEOMETRY = "geometry"
+LOCATION = "location"
+LATITUDE = "lat"
+LONGITUDE = "lng"
+
+
+class Geocoding:
+    def __init__(self):
+        key = os.environ.get(GOOGLE_API_KEY_ENV)
```
Comment (fparga) [inline action-server/covidflow/utils/geocoding.py:19]: Why not use `os.environ[GOOGLE_API_KEY_ENV]` if you raise when it's `None` anyways?

File: action-server/tests/utils/test_geocoding.py [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+
+class GeocodingTest(TestCase):
+    def test_missing_environment_key(self):
+        with self.assertRaises(expected_exception=Exception):
+            Geocoding()
+
+    @patch.dict("os.environ", ENV)
+    @patch("covidflow.utils.geocoding.googlemaps")
+    def test_get_geolocation_empty(self, mock_gogolemaps):
```
Comment (fparga) [inline action-server/tests/utils/test_geocoding.py:18]: `gogolemaps`? Hahaha

File: action-server/covidflow/utils/geocoding.py (hunk 2 above)
Comment (mathieubergeron) [reply action-server/covidflow/utils/geocoding.py:19]: right.

**Pull request merged**